pulsar-discovery-service/src/test/java/org/apache/pulsar/discovery/service/DiscoveryServiceTest.java

```diff
@@ -245,6 +248,9 @@ private void addBrokerToZk(int number) throws Exception {
         ZooKeeperChildrenCache availableBrokersCache = (ZooKeeperChildrenCache) field
                 .get(service.getDiscoveryProvider().localZkCache);
         availableBrokersCache.reloadCache(LOADBALANCE_BROKERS_ROOT);
+
+        Awaitility.await().atMost(10, TimeUnit.SECONDS).until(()
```

Comment:
I've been wondering if we should get rid of `.atMost(10, TimeUnit.SECONDS)` parts and simply rely on the [the Awaitility defaults](https://github.com/awaitility/awaitility/wiki/Usage#defaults). The default value for atMost is 10 seconds.

Reply:
okay, makes sense